src/main/views/lists/tabs/contactTypes/editContactType.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set isNewContactType = true %}
+{% if contactType.id %}
+  {% set isNewContactType = false %}
+{% endif %}
+
+{% if isNewContactType %}
+  <h2 class="govuk-heading-l">Add New Contact Type</h2>
+{% else %}
+  <h2 class="govuk-heading-l">Editing Contact Type: {{contactType.type}}</h2>
+{% endif %}
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+{% if fatalError == false %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        {{ govukInput({
+          label: {
+            text: "Name"
+          },
+          id: "ct-type",
+          name: "contactType[type]",
+          value: contactType.type,
```

Comment:
Should this be contactType.name? Or is type also the name? Confusing terminology but it's fine if it's already understood by everyone to be the case. May want to check it's also consistent name wise with what we store on the db

Reply:
currently api model model is set to use type